react/components/RequestOrganizationForm.tsx

```diff
@@ -178,8 +178,8 @@ const RequestOrganizationForm: FC = () => {
     setAddressState(() => addValidation(getEmptyAddress(country)))
   }
 
-  const [updateOrganizationRequest] = useMutation(UPDATE_ORGANIZATION_REQUEST)
-  const [updateOrganization] = useMutation(UPDATE_ORGANIZATION)
+  // const [updateOrganizationRequest] = useMutation(UPDATE_ORGANIZATION_REQUEST)
+  // const [updateOrganization] = useMutation(UPDATE_ORGANIZATION)
```

Comment:
Can we remove all these commented out lines?

Reply:
100%, on it